json-spec/extensions/stac-eo-spec.md

```diff
@@ -0,0 +1,60 @@
+# STAC EO Extension JSON Spec
+
+This document explains the fields of the STAC Earth Observation (EO) Extension to a STAC `Item`. EO data is considered to be data that represents a snapshot of the earth for a single date and time. It could consist of multiple spectral bands in any part of the electromagnetic spectrum. Examples of EO data include sensors with visible, short-wave and mid-wave IR bands (e.g., the OLI instrument on Landsat-8), long-wave IR bands (e.g. TIRS aboard Landsat-8), as well as SAR instruments (e.g. Sentinel-1).
+
+## EO Extension Description
+These are the additional fields to appear under a STAC `Item`, as well as the items's assets field and `Collection` level metadata.
+
+### `Item` additions
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| eo:collection             | string                      | Collection Name       | The name of the STAC `Collection` this `Item` belongs to |
+| eo:crs     | reference system    | ref system             | CRS of the datasource in full WKT format. null if no crs
+| eo:cloud_cover     | number (optional)   | Cloud Cover Pct    | Percent of cloud cover | 
+| eo:off_nadir      | number (optional)   | Off nadir    | The viewing angle measured from nadir, in degrees
+| eo:azimuth      | number (optional)   | Azimuth    | The viewing azimuth angle, in degrees
```

Comment:
Sign convention should also be defined as each provider might use a difference reference system.

Reply:
+1 to sign convention for sure.

Reply:
sign doesn't matter for an off_nadir angle, it would just be 0-90 degrees...or, I suppose 0-180 degrees if you want to allow for drones looking above the horizon (such as in mountainous regions).
The direction of that off_nadir angle would be determined by the look azimuth angle so that should probably be specified...Clockwise from North?   Off hand I have no idea what would be the best choice here based on providers... @jeffnaus ? @osialr ?